File: easy/cooling/cooling.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

# started at: Tuesday, 26. November 2013 10:54PM
import sys

iterations = int(sys.stdin.readline())

for iteration in xrange(iterations):
    pies = int(sys.stdin.readline())

    weights = sorted(map(int,sys.stdin.readline().split()))
    weight_limits = sorted(map(int,sys.stdin.readline().split()))

    count = 0
    for weight_limit in weight_limits:
        if weights[count]<=weight_limit:
            count+=1
    print count

    

File: easy/cooling/cooling.c
/*
    started at: Tuesday, 26. November 2013 10:54PM
*/

#include <stdio.h>
#include <stdlib.h>

int compare (const void * a, const void * b)
{
  return ( *(int*)a - *(int*)b );
}

int main(int argc, char const *argv[])
{
    int iterations = 0;
    scanf("%d",&iterations);

    int pies = 0;
    int count = 0;

    for (int i = 0; i < iterations; ++i)
    {   
        count = 0;
        scanf("%d",&pies);

        int* weights = (int *) calloc(pies,sizeof(int));
        int* weight_limits = (int *) calloc(pies,sizeof(int));

        for (int j = 0; j < pies; ++j)
        {
            scanf("%d",&weights[j]);
        }
        for (int k = 0; k < pies; ++k)
        {
            scanf("%d",&weight_limits[k]);
        }

        qsort(weights,pies,sizeof(int),compare);
        qsort(weight_limits,pies,sizeof(int),compare);

        for (int t=0; t<pies; ++t) 
        {
            if (weights[count]<=weight_limits[t])
            {
                count++;
            }
        }
        printf("%d\n",count);

    }

    return 0;
}